platform/feature-api/api/src/pod-api/default.ts

```diff
@@ -29,12 +29,27 @@ export const DEFAULT_POD_RUNTIME_VERSION = "podjs-default-version";
  * The [[PolyOut]] interface. See [[PolyOut]] for the description.
  */
 export class DefaultPolyOut implements PolyOut {
+    /**
+     * It creates a new instance of the class [[DefaultPolyOut]].
+     * @param {IFs["promises"]} fs - IFs["promises"]
+     */
     constructor(public readonly fs: IFs["promises"]) {}
 
+    /**
+     * It reads the file of the `path` given and returns its buffer.
+     * @param {string} path - The path to the file you want to read.
+     * @returns A promise that resolves to a buffer.
+     */
```

Comment:
Is it possible to somehow inherit the JSDoc of the {{PolyOut}} interface for all the overwritten stuff? Otherwise we'd have the docs duplicated.